### ISSUE TITLE
Future backlogs

### ISSUE THREAD

**Issue by gulprun** [China-Digital-Times-CDT/nft#8]
# UI 
- https://www.themarginalian.org/2014/10/24/how-to-do-nothing-with-nobody-all-alone-by-yourself/ 

# UX 

# System

**Comment by gulprun** [comment]
We are going to implement the detailed NFT(CDT) trading page next within MVP7

![Image](https://user-images.githubusercontent.com/33362680/184565543-aa9610fc-863b-429e-9875-423b4c298207.png)
![Image](https://user-images.githubusercontent.com/33362680/184565544-5fdd8d41-ffda-4e69-95c0-b5af76633f16.png)
![Image](https://user-images.githubusercontent.com/33362680/184565542-8834bbe7-d224-4661-8355-554175520524.png)
![Image](https://user-images.githubusercontent.com/33362680/184565545-9607383e-a048-48b4-9112-71a9e6275d39.png)

**Comment by gulprun** [comment]
The current draft of the trade page is not fully following the product idea above, 
1. The trade sub-page should be simpler than the main page, giving more space to focus on trading content
2. The price listed are the asking price from share holders, at the day one, there's no ownership yet, so it's 1984 with $1/each
3. user can choose amount from 1 to 1984 at day one, or less once some were purchased. 
![Image](https://user-images.githubusercontent.com/33362680/185026957-6e8c7f49-9e92-4571-9607-3df26b118dd4.png)

**Comment by gulprun** [comment]
the payment process is now working, although still very draft, we can improve the UX to make it smoother. Thanks.

**Comment by gulprun** [comment]
To be honest, such dialogue is quite horrible to users, can we find a better way to confirm the reception of the payment? And what's the changes to the shares in the backend? 


![Image](https://user-images.githubusercontent.com/33362680/185528916-93d28dda-28d7-4104-a012-70a84f625457.png)

**Comment by rkicoach** [comment]
I will use beautiful dialog box for confirming payment
and once it is paid, it is displaying total shares to dashboard page

**Comment by gulprun** [comment]
For MVP8, we need to have such user experiences as shown below(partially mockup but mostly connect to what we have equipped with like #17 ):

- Login  (Scan LNURL and get new users or back to registerred user if exists)
- Invited (Scan LNURL and get Sat into account)
- Wallet (Generate invoice with amount and deposit by scanning QR code, or withdraw amount to an invoice)
- Mockups at this moment: Settings/Collections(own shares from trading part, TBD), up next is to generate invitation links and possible to register through "Invited" page, 

![Image](https://user-images.githubusercontent.com/33362680/186308710-bd10708a-39e2-4e25-b602-bce4f9d6e17f.png)

**Comment by gulprun** [comment]
For now, what we've seen it's all mockups except the isolated LN-Auth part, can we post questions here beforehand next time if there're any unclear requirements. We have to move the unfinished parts to next MVP apparently.

**Comment by rkicoach** [comment]
for deposit/withdraw function, I think it is realated with wallet address.
First, in my opinion, we must implement wallet address.
Let's discuss how we implement wallet address more

**Comment by gulprun** [comment]
> for deposit/withdraw function, I think it is realated with wallet address.

not really, deposit will create an invoice and withdraw will scan or input an invoice, it's general process. Once again, please refer to stacker.news.

**Comment by gulprun** [comment]
summary of MVP9:
- the discussed scope has been a consensus
- withdraw/deposit function can work, with confirmed reliability by testing two different accounts  
- However, the new finding is CI/CD won't work as @rkicoach said: `but right now we need to run server every time after updating some code`, the CI/CD is reopened https://github.com/China-Digital-Times-CDT/nft-backend/issues/3 accordingly
- the github binding can't distinguish user using different wallets(on the same machine), after a `null` name bug fix
